### ISSUE TITLE
fix shit

### ISSUE THREAD

**Issue by deavmi** [deavmi/BoaGen#26]

**Comment by deavmi** [comment]
Damnit. Had to contact GitHub for this :rage:

**Comment by Walkman100** [comment]
You had to contact github??? Have they responded? Did they fix it?

**Comment by deavmi** [comment]
Idk. I have to check. I will explain later.

**Comment by deavmi** [comment]
I fact I will write a document about it.

**Comment by deavmi** [comment]
* **Infact**

**Comment by Walkman100** [comment]
:unamused: Why don't you just edit your comment...

Also:
> Have they responded? Did they fix it?

**Comment by deavmi** [comment]
Okay. Nevermind. It's working correctly, all it was was a siple thing, check out my Pjuu post, what @Joe said ws correct, it was just caching on the GitHub server's. all is good now. :smile:

**Comment by deavmi** [comment]
I will close this issue tomorrow. When I cofirm all of this.

**Comment by Walkman100** [comment]
Well what was even the problem??? I still don't know...

**Comment by deavmi** [comment]
Nope. It wasn't. I created this for fuck all reason . Lol. https://github.com/deavmi/The-deletion-problem-on-GitHub . Lol
